src/main/java/org/folio/linked/data/mapper/dto/monograph/work/sub/WorkSubResourceMapperUnit.java

```diff
@@ -12,4 +22,21 @@ default Set<Class<?>> supportedParents() {
     return Set.of(WorkRequest.class, WorkResponse.class);
   }
 
+  default Resource fetchResource(Identifiable identifiable, Function<String, Optional<Resource>> fetchByIdFunction,
```

Comment:
FYI: We may have to move this method to one level up (to SingleResourceMapperUnit) or to a util class in future. 
Reason is because, Instance also can have contributors. So, AgentMapperUnit have to inherit from SingleResourceMapperUnit in future.

Reply:
done